### DISCUSSION TITLE
error message while installing foundry

### DISCUSSION THREAD

**Discussion by WaleedMaghraby** [Cyfrin/foundry-full-course-cu#161] [Q&A] [answered]
Hi, Im facing this error message when installing foundry on my windows , VS code with WSL, 
error message is "" curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to foundry.paradigm.xyz:443""
command "" curl -L https://foundry.paradigm.xyz | bash""

anyone could give a hand as i give up

**Comment by cromewar** [comment, marked as answer]
Hello @WaleedMaghraby try with this:

```bash
curl -k -L https://foundry.paradigm.xyz | bash
```

**Reply by YourDailyBlockchain** [reply]
Awesome, thanks - had a similar issue - I think I have it now.

**Comment by WaleedMaghraby** [comment]
thanks @cromewar 
Appreciated.
